### PR TITLE
Added request to backend when subscribe to newsletter

### LINKED ISSUE #26: Front newsletter
Todo : 
1. Call to the backend with the user email. 
2. Create a OK-popup.

### PR DESCRIPTION
**Pull request by BourgoisMickael**
resolve #26

### CODE REVIEW

File: templates/index.html.hbs [hunk 1]
```diff
@@ -32,6 +32,31 @@
       Author: BootstrapMade.com
       License: https://bootstrapmade.com/license/
     ======================================================= -->
+
+    <script>
+      function requestBackend() {
+        var email = document.getElementById("email-subscribe").value;
+
+        var http = new XMLHttpRequest();
+        var url = 'http://127.0.0.1:8000/newsletter';
```
Comment (Arignir) [inline templates/index.html.hbs]: You shoudln't use the domain name and use absolute path insead `/newsletter`

File: templates/index.html.hbs [hunk 2]
```diff
@@ -32,6 +32,31 @@
...
+        http.open('POST', url, true);
+
+        //Send the proper header information along with the request
+        http.setRequestHeader('Content-type', 'application/json');
+
+        http.onreadystatechange = function() {//Call a function when the state changes.
+            if(http.readyState == 4) {
+              if (http.status == 201) {
```
Comment (Arignir) [inline templates/index.html.hbs]: Please use 4-spaces indentation

File: templates/index.html.hbs (hunk 1 above)
Comment (BourgoisMickael) [reply templates/index.html.hbs]: done

File: templates/index.html.hbs (hunk 2 above)
Comment (BourgoisMickael) [reply templates/index.html.hbs]: fixed

**Pull request merged**